src/main/com/fulcrologic/rad/database_adapters/sql.clj

```diff
@@ -105,26 +109,27 @@
             auto-create-missing?
             (do
               (log/info "Automatically trying to create SQL schema from attributes.")
-              (jdbc/execute! db (automatic-schema schema all-attributes))))
+              (jdbc/execute! (:datasource db) (automatic-schema schema all-attributes))))
```

Comment:
Really?  They changed the interface away from the map of attributes?  In that case I'd rather port it around so we're passing the data source around instead of the map.  I guess we can figure that out when we rework things a bit to support top-level transactions controlled at the begin/end of a parser run.

Reply:
Yeah passing the datasource around directly is an option. jdbc.next offers a Executable protocol. I was thinking to reify our own db record with it so we can also get logging around every query

Reply:
Yeah, I'd rather not see that extraction on every sql stmt